app/routes/matador.tsx

```diff
@@ -1,18 +1,12 @@
 import type { ColorScheme } from "@mantine/core";
-import {
-  ColorSchemeProvider,
-  MantineProvider
-} from "@mantine/core";
+import { ColorSchemeProvider, MantineProvider } from "@mantine/core";
 import type { LinksFunction, LoaderFunction } from "@remix-run/node";
 import { Outlet, useCatch, useLoaderData } from "@remix-run/react";
 import { useEffect, useState } from "react";
-import { BiInfoCircle } from "react-icons/bi";
-import { BsArrowsAngleContract } from "react-icons/bs";
-import { DiRedis } from "react-icons/di";
-import type { NavbarItem } from "~/lib/matador/components";
-import { NavBar } from "~/lib/matador/components/navbar";
+import { NavBar, ErrorFallback } from "~/lib/matador/components/index";
 import { themeKeyLocalStorage } from "~/lib/matador/constants";
 import { getMatadorVersion } from "~/lib/matador/index.server";
+import Navigations from "~/lib/matador/navigations";
```

Comment:
remember to refactor this like above

Reply:
Fixed with f089a705ecee1a85e45b3512238c5af13eef525a